cmd/oas-expand/expand.go

```diff
@@ -0,0 +1,41 @@
+// CLI utility that expands OAS file to reduce init time.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/tetafro/oas2"
```

Comment:
should import this repo and not your fork :wink:

Reply:
oops

Reply:
fixed